docs/code-signing.rst

```diff
@@ -0,0 +1,10 @@
+Code signing policy
+===================
+
+The code used by the DebOps Project and commited in its repositories is
+expected to be signed by a valid GPG key of a DebOps Developer or DebOps
+Contributor. This should allow for secure code authentication and a valid audit
```

Comment:
> But even there, if your laptop with your unprotected SSH private key and your GPG key is gone, even a commit signature won't protect the code.


@ganto That is something I have been thinking about for which I will hopefully implement a solution someday in the future. To address this, I plan to swap out GPG subkeys more often and to destroy the GPG subkey’s private key after it has been expires so that even I could not fake a signature after a certain time. Obviously I would need to rethink my backup strategy for my GPG keys for this to work.